.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,10 @@ Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 - [ ] documentation is changed or added
 - [ ] commit message follows commit guidelines
 
+<!-- If this is your first contribution to Node.js, please also check the
```

Comment:
Possibly. I think in practice not many people read the terms and conditions before checking the box. Not a big deal though.